shared/chat/conversation/messages/cards/team-journey/util.tsx

```diff
@@ -37,10 +38,14 @@ function renderWelcomeMessage(
   cannotWrite: boolean
 ): React.ReactNode {
   return (
-    <Kb.Markdown smallStandaloneEmoji={false} lineClamp={3} selectable={false}>
+    <Kb.Markdown smallStandaloneEmoji={false} lineClamp={3} selectable={false} paragraphTextClassName="text_BodySmall" style={welcomeStyle}>
       {removeWhitespaceOnlyLines(computeWelcomeMessageText(message, cannotWrite))}
     </Kb.Markdown>
   )
 }
 
+const welcomeStyle = {
```

Comment:
this should be defined inside a new `Styles.styleSheetCreate`

Reply:
Fixed.

Reply:
```typescript
const styles = Styles.styleSheetCreate(() => ({
  welcomeStyle: {
    ...
  },
} as const))

Reply:
Fixed.